### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by SHeimlich**
Some small changes to the README to improve readability and integrate better with Glitch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,22 +136,24 @@ The `render` function is given an opaque reference to a list of fonts. This is
...
+to templates for rendering.
+
+In the example above, the `render` function return an object with
 a key called `myfonts` with two properties: `models` and `$template`. _Models_
-points to the input `fonts` opaque reference, and `$template` refers to a
-`<template>` element which will be stamped down for each font in the
-input. The `<div>{{myfonts}}</div>` triggers the application of the template to
-the opaque font list. (_Note_ that in templates, double curly brace `{{x}}` is
-template interpolation.)
+points to the input `fonts` opaque reference. Similarly `$template` points to `font_T`
```
Comment (shaper) [inline README.md:143]: `font_t` (lower-case last T) here and below.

File: README.md [hunk 2]
```diff
@@ -162,21 +168,24 @@ This just maps the fields of the font directly to the output without change, but
...
 ```
 
-Here you see what the `{{name}}` and `{{displayStyle}}` references earlier were
-pointing to. There are also two new fields,
-`key` and `sortKey`. After the _decorator_ has been applied to the every item in
+Here you see the `{{name}}` and `{{displayStyle}}` references from earlier are
+pointing to `fullName` and CSS formatting respectively. There are also two new fields,
+`key` and `sortKey`.
+
+After the _decorator_ has been applied to the every item in
```
Comment (shaper) [inline README.md:188]: "to every" (rm 'the')

File: README.md [hunk 3]
```diff
@@ -203,15 +212,19 @@ file.
...
+click the "Simple Decorator" button on the right to see the results of this
+code.
 
 ## Filtering
 
-Sometimes you want to apply a filter to a list in order to a subset. This is
-supported, and filters are applied after _decorator_s and can utilize that data
-as well. As an example, here's a filter for only _bold_ fonts to be shown.
+Sometimes you want to apply a filter to a list to obtain a subset. This is
+supported, and filters are applied after _decorators_. As a reult,
```
Comment (shaper) [inline README.md:222]: reult -> result

File: README.md [hunk 4]
```diff
@@ -203,15 +212,19 @@ file.
...
+decorators can use the data from the decorator. As an example, here's a
```
Comment (shaper) [inline README.md:223]: Is this a typo, do we mean decorators can use the data from the filter?

File: README.md [hunk 5]
```diff
@@ -225,15 +238,17 @@ render({fonts}) {
       models: fonts,
       $template: 'font_t',
       decorator: `decorator`,
```
Comment (shaper) [inline README.md:240]: Should we update this (and elsewhere, to be consistent) to `myDecorator` like you did above? Or move the edit to `myDecorator` back to just `decorator`? Either seems ok, but consistency would be good. I get why you probably edited it to `myDecorator, to be clearer/less confusing.

File: README.md [hunk 6]
```diff
@@ -243,13 +258,13 @@ piece of the puzzle is to use event handlers to allow a font to be clicked on
...
-to a typical _DOM UI Event_ name.
+template. These attributes always begin with `on-` and have a suffix corresponding
+to a typical _DOM UI Event_ name. (For example: `on-click`.)
 
 The difference between event handling in our templates and in regular Web applications
-is that the DOM event data is scrubbed, and you are given an `eventlet` and
-a `key` or `value` of your own choice supplied in the template. Let's look at a
-simple example:
+is that the DOM event data is scrubbed, and instead the referenced function will
+recive an `eventlet` and a `key` or `value` of your own choice supplied in the template.
```
Comment (shaper) [inline README.md:266]: recive -> receive

File: README.md [hunk 7]
```diff
@@ -363,14 +382,14 @@ decorator(modelItem, inputs, state)
...
-are specific to the *ArcsJs* system. _Inputs_ are data provided by a ArcsJs _
+are specific to the _ArcsJs_ system. _Inputs_ are data provided by a ArcsJs _
 Store_ connected to your Particle by the _Recipe_. `state` is an object
 maintained by the ArcsJs system that allows you to store and reference global
 data. The ArcsJs system is reactive, so anytime `inputs` or `state` changes,
 your particle is re-executed to update the UI.
 
 Ordinarily, `state` is globally mutable and readable, however this would be
-dangerous in a *Non Permissive Particle* because it could accumulate global font
+dangerous in a _Non Permissive Particle_ because it could accumulate global font
```
Comment (shaper) [inline README.md:392]: We hyphenate this ("Non-Permissive Particle") several times earlier in the doc, we should be consistent one way or the other.

File: README.md [hunk 8]
```diff
@@ -514,28 +532,27 @@ render({fonts}) {
...
+onFavorite({eventlet: {value}}) {
     value.favorite = true;
 },
 ```
 
 We can also add an `onUnfavorite` method.
 
 ```
-onFavorite({eventlet: {value}}) {   
+onFavorite({eventlet: {value}}) {
```
Comment (shaper) [inline README.md:543]: This looks like a typo (present before your PR), and should be `onUnfavorite` I think.

File: README.md [hunk 9]
```diff
@@ -2,29 +2,31 @@
...
+- Only a single datum, the selected font, may be returned to the host page.
 
 Despite those restrictions, font pickers can be built with rich functionality
 including arbitrary grouping, mixing of local and web fonts, search and sort
 functionality, and even the ability to create and store new per-font metadata
 like a personal rating or favoriting mechanism.
 
 The full system is very flexible, but for the purposes of this tutorial, we will
-only be modifying **Recipes** to customize the system.
+only be modifying [**Recipes**](https://github.com/project-oak/arcsjs-chromium#recipes)
```
Comment (shaper) [inline README.md:28]: I think this is fine but just to mention, we do also have an appendix/section at bottom of the code lab that describes these things. We could link to those here, and link those to these for more -- or keep this. Just raising for re-consideration.

**Review by shaper — APPROVED** [review]

**Comment by cromwellian** [comment]
Just gonna merge this and address the comments in separate PR.

**Pull request merged**